examples/drdonut-nextjs/utils/callFunctions.ts

```diff
@@ -53,33 +53,31 @@ export async function startCall(callbacks: CallCallbacks): Promise<() => void> {
 
     await initUVSession();
 
-    let state: UltravoxSessionState;
     if (UVSession) {
-      state = UVSession.joinCall(joinUrl);
-      console.log('Session status:', state.getStatus());
+      UVSession.joinCall(joinUrl);
+      console.log('Session status:', UVSession.status);
     } else {
       return () => {};
     }
 
     const statusChangeListener = (event: any) => {
-      callbacks.onStatusChange(event.state);
+      callbacks.onStatusChange(UVSession?.status);
     };
 
     const transcriptChangeListener = (event: any) => {
-      let te: UltravoxSessionStateChangeEvent = event;
-      console.log(te.transcripts);
-      callbacks.onTranscriptChange(te.transcripts);
+      console.log(UVSession?.transcripts);
+      callbacks.onTranscriptChange(UVSession?.transcripts);
     };
 
-    state.addEventListener('ultravoxSessionStatusChanged', statusChangeListener);
-    state.addEventListener('ultravoxTranscriptsChanged', transcriptChangeListener);
+    UVSession.addEventListener('status', statusChangeListener);
```

Comment:
Since we're making changes anyway, best practice would be to add event listeners before joinCall since you'll miss some startup events otherwise.

Reply:
done